scripts/utils/create_package_template.py

```diff
@@ -427,7 +431,7 @@ def create_template(
         )
 
     with open(os.path.join(tools_path, UNINSTALL_TEMPLATE_NAME), "w") as f:
-        f.write(uninstall_template.format(tool_name=tool_name, category=category))
+        f.write(uninstall_template.format(tool_name=tool_name, ))
```

Comment:
[nitpick] I assume you have tested and this does not make the script failing, otherwise it is not a nitpick :wink: 
```suggestion
        f.write(uninstall_template.format(tool_name=tool_name))
```

Reply:
I realized as well, the script worked but anyways I removed it